.storybook/theme.ts

```diff
@@ -53,15 +53,15 @@ export const darkTheme = create({
   barSelectedColor: tokens["color-base-blue--100"],
 
  // UI
- appBg: '#ddffff',
- appContentBg: 'var(--color-surface)',
- appPreviewBg: 'var(--color-surface)',
- appBorderColor: '#585C6D',
- appBorderRadius: 4,
+  appBg: tokens["color-base-blue--1000"],
```

Comment:
As a follow up it would be good to use semantic tokens here!

Reply:
sounds good - I can probably do that as side desk work within this sprint